src/renderer/features/wallet-connect-wallet/components/WalletIcon.tsx

```diff
@@ -8,15 +8,16 @@ import { walletConnect } from '../model/connect';
 
 type Props = {
   wallet: Wallet;
+  className?: string;
   size: number;
 };
 
-export const WalletIcon = ({ wallet, size }: Props) => {
+export const WalletIcon = ({ wallet, size, className }: Props) => {
```

Comment:
Overall direction for new components and old that's being refactored - don't use `className` and apply styles with wrappers in place of usage